.github/workflows/ci.yml

```diff
@@ -44,9 +44,41 @@ jobs:
       run: make build
     - name: Check clean repository
       run: make check-clean-work-tree
-
-  test-race:
+  lint-windows:
```

Comment:
1. Why not `runs-on: windows-lastest`?
2. Maybe we could try using `strategy: matrix` in the `lint` job like for `test-race`

Reply:
Good point, wasn't expecting it to work, looks like it's green ✅